packages/modelviewer.dev/data/docs.json

```diff
@@ -5,65 +5,65 @@
       {
         "name": "src",
         "htmlName": "src",
-        "description": "<p>The URL to the 3D model. This parameter is required for <span class=\"attribute\">&lt;model-viewer&gt;</span> to display. Only <a href=\"https://github.com/KhronosGroup/glTF/tree/master/specification/2.0\">glTF</a>/<a href=\"https://github.com/KhronosGroup/glTF/tree/master/specification/2.0#glb-file-format-specification\">GLB</a> models are supported, see <a href=\"https://github.com/google/model-viewer#supported-formats\">Supported Formats</a>.</p>",
+        "description": "The URL to the 3D model. This parameter is required for <span class=\"attribute\">&lt;model-viewer&gt;</span> to display. Only <a href=\"https://github.com/KhronosGroup/glTF/tree/master/specification/2.0\">glTF</a>/<a href=\"https://github.com/KhronosGroup/glTF/tree/master/specification/2.0#glb-file-format-specification\">GLB</a> models are supported, see <a href=\"https://github.com/google/model-viewer#supported-formats\">Supported Formats</a>.",
         "default": [
           "String",
-          "any legal filepath",
-          "empty string"
+          "empty string",
+          "any legal filepath"
         ]
       },
       {
         "name": "alt",
         "htmlName": "alt",
-        "description": "<p>Configures the model with custom text that will be used to describe the model to viewers who use a screen reader or otherwise depend on additional semantic context to understand what they are viewing.</p>",
+        "description": "Configures the model with custom text that will be used to describe the model to viewers who use a screen reader or otherwise depend on additional semantic context to understand what they are viewing.",
         "default": [
           "String",
-          "any description",
-          "empty string"
+          "empty string",
+          "any description"
         ]
       },
       {
         "name": "poster",
         "htmlName": "poster",
-        "description": "<p>Displays an image instead of the model, useful for showing the user something before a model is loaded and ready to render. If you use a poster with transparency, you may also want to set --poster-color to transparent so that the background shows through.</p>",
+        "description": "Displays an image instead of the model, useful for showing the user something before a model is loaded and ready to render. If you use a poster with transparency, you may also want to set --poster-color to transparent so that the background shows through.",
         "links": [
           "<a href=\"../examples/loading\">Related Examples</a>"
         ],
         "default": [
           "String",
-          "link to image",
-          "empty string"
+          "empty string",
```

Comment:
I feel like there's a more standard way for the show sidebar icon to look; maybe Ricardo will have an idea. Are you comfortable enough with the design to get feedback from him once this is merged?

Reply:
yeah, he's on vacation, though, right?

Reply:
Yeah, but he said he wouldn't mind looking through it and giving some feedback. Not on the code, but in terms of playing with the deployed site.

Reply:
yeah that would be nice, the two things I'm still not sure about are the best way to show a section is active in the sidebar (something more than an underline) and possibly a font change to the sidebar